packages/contracts-bedrock/src/L2/L1Block.sol

```diff
@@ -75,12 +75,6 @@ contract L1Block is ISemver {
         return "1.6.2";
     }
 
-    /// @notice Returns the gas paying token, its decimals, name and symbol.
-    function gasPayingToken() public pure returns (address addr_, uint8 decimals_) {
```

Comment:
We need to be careful with removing this because Unichain is on a version of `WETH` that depends on this. If we do decide to remove this, then we need to also upgrade the version of `WETH` that Unichain is using

Reply:
cc @Joxess - if i recall, there is a doc someplace that mentions this